pkgs/servers/mail/archiveopteryx/default.nix

```diff
@@ -0,0 +1,18 @@
+{ stdenv, fetchurl, openssl, perl, zlib }:
+with builtins;
+let
+  version = "3.2.0";
+in stdenv.mkDerivation {
+  name = "${baseNameOf ./.}-${version}";
```

Comment:
How strong is the preference of `rec` over `let ... in`? I kinda dislike `rec` :)
The rest has been updated as you said.